src/node_debug_options.cc

```diff
@@ -94,18 +94,18 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
   if (option_name == "--inspect") {
     debugger_enabled_ = true;
     enable_inspector = true;
-  } else if (option_name == "--inspect-brk") {
+  } else if (option_name == "--inspect-brk" || option_name == "--debug-brk") {
     debugger_enabled_ = true;
     enable_inspector = true;
     wait_connect_ = true;
-  } else if ((option_name != "--debug-port" &&
-              option_name != "--inspect-port") ||
-              !has_argument) {
-    // only other valid possibility is --inspect-port,
-    // which requires an argument
+  } else if (option_name == "--inspect-port" || option_name == "--debug-port") {
```

Comment:
Personally, I would rather `--inspect-port=9999 --inspect=9876 --inspect-brk-8888` result in an error because there's obviously no way to correctly interpret or implement the users intent. Essentially, if the `host:port` is specified multiple times, there should be an error.